tool/gravity/cli/rpcagent.go

```diff
@@ -544,6 +545,28 @@ func getGravityPackage() loc.Locator {
 	}
 }
 
+func (r deployAgentsRequest) String() string {
```

Comment:
I added this to avoid the request logging binary data so we only log the relevant parts.